js/tuxexchange.js

```diff
@@ -0,0 +1,533 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ArgumentsRequired, NotSupported, AuthenticationError, InsufficientFunds, OrderNotFound, BadRequest } = require ('./base/errors');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class tuxexchange extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'tuxexchange',
+            'name': 'Tux Exchange',
+            'countries': ['CA'],
+            'version': '',
+            'accounts': undefined,
+            'accountsById': undefined,
+            'hostname': 'tuxexchange.com/api',
+            'has': {
+                'CORS': false,
+                'fetchCurrencies': true,
+                'fetchTicker': false, // Can be emulated on fetchTickers if necessary
+                'fetchTickers': true,
+                'fetchTrades': true,
+                'fetchTradingFees': false,
+                'fetchBalance': true,
+                'createOrder': true,
+                'cancelOrder': true,
+                'fetchOpenOrders': true,
+                'fetchMyTrades': true,
+                'fetchDepositAddress': true,
+                'fetchWithdrawals': true,
+                'fetchDeposits': true,
+                'fetchClosedOrders': false,
+                'fetchL2OrderBook': false,
+                'fetchOHLCV': false,
+                'fetchOrder': false,
+                'editOrder': false,
+                'fetchTransactions': false,
+                'fetchLedger': false,
+                'withdraw': false,
+            },
+            'urls': {
+                'logo': 'https://www.tuxexchange.com/images/kittytrade.png',
+                'api': 'https://www.tuxexchange.com/api',
+                'www': 'https://www.tuxexchange.com',
+                'doc': 'https://www.tuxexchange.com/docs',
+                'fees': 'https://www.tuxexchange.com/faq',
+            },
+            'api': {
+                // All methods are passed in as query params
+                'public': { 'get': [''] },
```

Comment:
We still need to point them out here instead of hardcoding them inside the methods directly.